### PR TITLE
Change to sysConfig for BetaFlight 3.2 

### PR DESCRIPTION
**Pull request by DieHertz**
While studying a flight log from current BetaFlight master I have noticed yaw RC command showing `NaN (degrees/s)`.
Found and fixed some other renamed parameter along with yaw-related ones.

### CODE REVIEW

File: js/flightlog_parser.js [hunk 1]
```diff
@@ -198,11 +198,11 @@ var FlightLogParser = function(logData) {
         // standard logger.
 
         defaultSysConfigExtension = {
-            rcExpo:null,              	    // RC Expo
-            rcYawExpo:null,                 // Yaw Expo
-            thrMid:null,              	    // Throttle Mid Position
-            thrExpo:null,              	    // Throttle Expo
-            dynThrPID:null,                 // TPA
+            rc_expo:null,              	    // RC Expo
```
Comment (martinbudden) [inline js/flightlog_parser.js]: I wouldn't change these names, eg keep `rcExpo` etc and see changes below.

File: js/flightlog_parser.js [hunk 2]
```diff
@@ -417,13 +417,13 @@ var FlightLogParser = function(logData) {
                 that.sysConfig[fieldName] = parseInt(fieldValue, 10);
                 that.sysConfig.motorOutput[1] = that.sysConfig[fieldName]; // by default, set the maxMotorOutput to match maxThrottle
             break;
-            case "rcRate":
-            case "rcExpo":
-            case "rcYawExpo":
-            case "rcYawRate":
-            case "thrMid":
-            case "thrExpo":
-            case "dynThrPID":
```
Comment (martinbudden) [inline js/flightlog_parser.js]: Keep the above `case` statements. And then add statements below.

File: js/flightlog_parser.js [hunk 3]
```diff
@@ -564,15 +564,15 @@ var FlightLogParser = function(logData) {
             break;
             case "gyro.scale":
```
Comment (martinbudden) [inline js/flightlog_parser.js]: Add:
```
    case "rcRate":
          that.sysConfig.rc_rate = parseInt(fieldValue, 10);
          break;
    case "rcExpo":
        that.sysConfig.rc_expo = parseInt(fieldValue, 10);
        break;
    case "rcYawExpo":
        that.sysConfig.rc_expo_yaw = parseInt(fieldValue, 10);
        break;
    case "rcYawRate":
        that.sysConfig.rc_rate_yaw = parseInt(fieldValue, 10);
        break;
    case "thrMid":
        that.sysConfig.thr_mid = parseInt(fieldValue, 10);
        break;
    case "thrExpo":
        that.sysConfig.thr_expo = parseInt(fieldValue, 10);
        break;
    case "dynThrPID":
        that.sysConfig.tpa_rate = parseInt(fieldValue, 10);
        break;
```
to keep backward compatibility - so people can read old logfiles

**Comment by DieHertz** [comment]
Turns out there's a merged PR for CF BlackBox explorer, we just have to cherry-pick it. I have wasted my time :-(

**Pull request closed without merging**